src/renderer/containers/QuickStart/VerifyEmailStep.tsx

```diff
@@ -104,7 +105,7 @@ export const VerifyEmailStep: React.FC = () => {
         Check Verification
       </Button>
       <div css={classes.emailNotFoundContainer}>
-        Not finding email?{" "}
+        Not finding email? Check spam folder. Still missing?{" "}
```

Comment:
```suggestion
        Can't find the email? Check your spam folder. Still missing?{" "}
```
bit more verbose, but sounds better than what is currently there. could drop `verification` for brevity.